docs/guides-using-fake-timers.mdx

```diff
@@ -26,12 +23,10 @@ beforeEach(() => {
 })
 ```
 
-When using fake timers, you need to remember to restore the timers after your
-test runs.
-
-The main reason to do that is to prevent 3rd party libraries running after your
-test finishes (e.g cleanup functions), from being coupled to your fake timers
-and use real timers instead.
```

Comment:
And here, I would argue 3rd party cleanup functions are not the main reason (though a valid one). Restoring timers is necessary to prevent fake timers leaking into other tests, where you don't expect them (especially with parallel and sharded tests!). So I clarified both reasons without singling out either of them as the "main" reason